source/supported-hardware/apollo-lake-crb.rst

```diff
@@ -5,47 +5,22 @@ Apollo Lake CRB Boards
 
 .. note:: Intel Atom® Processor E3900 is formally known as |APL|.
 
-Board ID Assignments
+Supported Boards
 ^^^^^^^^^^^^^^^^^^^^^
 
-|SPN| supports the following board variations based on |APL|. Each board is assigned a unique platform ID by reading a set of GPIO pins (25, 26 and 30).
-
-  +-----------------+---------------+
-  |      Board      |  Platform ID  |
-  +-----------------+---------------+
-  |   Oxbox Hill    |       6       |
-  +-----------------+---------------+
-  |    Leaf Hill    |       7       |
-  +-----------------+---------------+
-  |  Juniper Hill   |       8       |
-  +-----------------+---------------+
-
-
-See :ref:`dynamic-platform-id` for more details.
-
-To customize board configurations in ``*.dlt`` file, make sure to specify ``PlatformId`` to the corresponding values for the board.
-
-See :ref:`configuration-tool` for more details.
-
-
-
-Debug UART
-^^^^^^^^^^^
-
-For |APL|, LPSS UART **Port 2** is the debug UART configured in |SPN|.
+|SPN| supports **Leaf Hill, Juniper Hill and Oxbow Hill** board variations of |APL|. 

```

Comment:
So Im assuming WHL/CFL will be added to the list of supported boards later?

Reply:
Yes, that will be a separate PR.